src/docs/spec-model.md

```diff
@@ -192,3 +192,47 @@ If a defined extension slot has a `type_hint` other than `http://www.w3.org/2001
 | http://www.w3.org/2001/XMLSchema#datetime | Implementations MAY check that the value is a date and time value in the ISO 8601 format (`yyyy-mm-ddThh:mm:ssTZ`) |
 
 Implementations MAY decide to recognise more types and to enforce type-specific constraints. For example, an implementation could recognise the type `http://www.w3.org/2001/XMLSchema#negativeInteger` and check that the value starts with a minus sign.
+
+## Versioning
+
+Starting from version 1.1 of the specification, the `MappingSet` class has an optional slot named `sssom_version` which indicates the version of the specification that the set declares itself to be compliant with.
+
+### Versioning rules
+
+The version number of the SSSOM specification has two components: a _major_ number _X_ and a _minor_ number Y, expressed as `X.Y`.
+
+A set that is compliant with a minor version _X.Y_ is also compliant with any minor version _X.Y+n_, for any value of _n_. The opposite is not true: a set compliant with a minor version _X.Y_ may not necessarily be compliant with a minor version _X.Y-n_.
+
+A set that is compliant with a major version _X_ may not be compliant with any other major version _X+n_ or _X-n_.
+
+Therefore, an implementation that is itself compliant with version _X.Y_ SHOULD always accept a set compliant with any version _X.Y-n_. It MAY reject outright a set compliant with any version _X.Y+n_ (more recent minor version), _X-n_ (older major version), or _X+n_ (more recent major version).
+
+In other words, the SSSOM specification guarantees backwards compatibility between two versions (in that a set compliant with an older version can be used with an implementation compliant with a newer version) only insofar as only the _minor_ version has changed.
+
+### Using the `sssom_version` slot
+
+When reading a SSSOM set:
+
+(A) If the set contains a `sssom_version` slot, implementations SHOULD check whether they recognize the indicated version as a supported version according to the rules in the previous section; if they don’t, they MAY reject the set outright.
+
+(B) If the set does not contain a `sssom_version` slot, it MUST be assumed to be compliant with version 1.0.
```

Comment:
This seems like an important piece of info; any chance it can be decorated a bit more to stand out?

Reply:
This is a specification document. Almost _everything_ in it is important for anyone wanting to develop SSSOM-compliant software. This particular piece of info is already highlighted with a MUST keyword, which is the RFC 2119 keyword that denotes the highest level of importance (MUST = “absolute requirement”).